tests/layer_tests/pytorch_tests/pytorch_layer_test_class.py

```diff
@@ -92,10 +92,20 @@ def _test(self, model, ref_net, kind, ie_device, precision, ir_version, infer_ti
             fw_res = (fw_res,)
 
         output_list = list(infer_res.values())
-        assert len(fw_res) == len(
-            output_list), f'number of outputs not equal, {len(fw_res)} != {len(output_list)}'
+
+        flatten_fw_res = []
+        for res_item in fw_res:
+            if res_item is None:
+                continue
+            if isinstance(res_item, list):
+                flatten_fw_res.extend(res_item)
+                continue
+            flatten_fw_res.append(res_item)
```

Comment:
Can we print a kind of warning informing that we are really flattening results from two nested levels and this can really be ambiguous (I mean for cases (x, [y, z]) and ([x, y], z). I believe that eventually we are going to fix tuple and list representation on outputs in a good way and such warning will help us to find all such cases or at least not to loose them completely.

Reply:
Or even better, put this warning to transformation itself as a reminder for users that output format is not what they have in the original model.

Reply:
I think this is acceptable for experimental code.

Reply:
so, sorry for stupid question, but how put warning to transformation?

Reply:
this will be enough?